### PR TITLE
return only schedules that have time segments.

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
amirite?

screenshot 2018-02-01 13 00 33
screenshot 2018-02-01 13 00 19

### CODE REVIEW

**Comment by sdunster** [comment]
@sjmcculloch this is on you - you're working on this stuff. Review, or take what you need and close it.

File: server/src/logic.js [hunk 1]
```diff
@@ -343,7 +345,17 @@ export const getHandlers = ({ models, creators: Creators }) => {
       users(group) {
         return group.getUsers();
       },
-      schedules(group) {
+      schedules(group, args) {
+        const { hasTimeSegments } = args;
+        console.log('MARK', hasTimeSegments);
```
Comment (sjmcculloch) [inline server/src/logic.js]: Can we remove this console.log

**Comment by OSPFNeighbour** [comment]
just confirming we still want this to be merged?

**Comment by sjmcculloch** [comment]
It's not strictly needed at the moment (the code does the filtering) - we will move this to a separate query eventually so either way is OK.

**Comment by sdunster** [comment]
its cool we can just leave this around for now.

**Pull request closed without merging**